components/_global-navigation.scss

```diff
@@ -93,21 +93,18 @@ $notification-counter-border-width: 1px;
 	&__icon-bell {
 		fill: $color-dark-blue-gray;
 		height: 18px;
-		vertical-align: middle;
 		width: 18px;
 	}
 
 	&__icon-dropdown {
 		fill: $color-dark-blue-gray;
 		height: 10px;
-		vertical-align: middle;
 		width: 10px;
 	}
 
 	&__content-bar {
 		border-top: $border-top-width solid $color-slate-gray;
 		color: $color-dark-blue-gray;
-		display: inline-block;
```

Comment:
I had a feeling it was needed here to make ellipsis work on longer translations.